src/common/playlists/playlist.css

```diff
@@ -462,6 +462,21 @@
 
 }
 
+.play-details .header-actions .action.counted {
+    display: flex;
+    width: auto;
+    height: auto;
+    text-decoration: none;
```

Comment:
`none` is default value for text-decoration. isn't it? Or it's being used to overwrite some parent property?

Reply:
i didnt make ui changes